include/dsVideoDevice.h

```diff
@@ -243,6 +247,8 @@ dsError_t dsGetSupportedVideoCodingFormats(intptr_t handle, unsigned int * suppo
 /**
  * @brief Gets the video codec information
  *
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ *
```

Comment:
Please update this from source point of view as well.  The description should clearly defined on what is expected from this API.

Reply:
Updated

Reply:
For the said API, the video codec is the input, while the description says it returns the "supported video codes"

Reply:
Can you come back on this whether we need this API or need to deprecate?

Reply:
As per the middleware team input, we cannot remove it because  it is used for diagnostic purpose.

Reply:
dsVideoCodecInfo_t  - Repharse the statement as below(This function not return supported video codecs)
"For source devices, this function returns the supported codec profile and levels for the input coding format."